packages/scandipwa/src/component/ProductReviews/ProductReviews.component.js

```diff
@@ -95,13 +98,15 @@ export class ProductReviews extends PureComponent {
         return (
             <>
                 { this.renderRatingSchema(rating_summary, review_count) }
-                <ProductReviewRating
-                  mix={ { block: 'ProductReviews', elem: 'SummaryRating' } }
-                  summary={ rating_summary }
-                />
+                { reviewRatings.length > 0 && (
+                    <ProductReviewRating
+                      mix={ { block: 'ProductReviews', elem: 'SummaryRating' } }
+                      summary={ rating_summary }
+                    />
+                ) }
                 <p block="ProductReviews" elem="SummaryDetails">
-                    { percent }
-                    <span>{ __('%s reviews', review_count || 0) }</span>
+                    { reviewRatings.length > 0 && <span>{ percent }</span> }
```

Comment:
Do you really need the empty `<span>` element here?

Reply:
If you going to style it, please give the `span` the implicit class name (block-element-etc).